test/truffle/upgradedContract.ts

```diff
@@ -0,0 +1,10 @@
+import fs from 'fs';
+
+const claimManagerContract = fs.readFileSync('./contracts/roles/ClaimManager.sol', 'utf8');
+
+// Creates a different copy of the ClaimManager contract
```

Comment:
Could this be added to help explain its purpose?
```suggestion
// Creates a v0.2 of the ClaimManager contract so that it can be used as an input to the upgrade test
// (i.e. the test needs a contract to upgrade to)
```